configure.ac

```diff
@@ -1520,11 +1522,52 @@ AS_IF([test x"$enable_spacetime" != "xyes" ],
     AC_MSG_CHECKING([whether to use libunwind])
     AS_IF([test x"$with_libunwind" = "xno"],
       [AC_MSG_RESULT([disabled])],
-      [AC_CHECK_HEADER([libunwind.h],
-        [AC_DEFINE([HAS_LIBUNWIND])
-        AC_MSG_RESULT([yes])],
-        [AC_MSG_RESULT([no])])
-      ])
+      [AS_IF([test x"$with_libunwind" = "x"],
+        [libunwind_requested=false
+        AC_MSG_RESULT([if available])],
+        [libunwind_requested=true
+        AC_MSG_RESULT([requested])
+        AS_IF([test x"$with_libunwind" != "xyes"],
+          [AS_IF([test x"$LIBUNWIND_INCLUDE_DIR" = "x"],
+            [LIBUNWIND_INCLUDE_DIR="$with_libunwind/include"])
+          AS_IF([test x"$LIBUNWIND_LIB_DIR" = "x"],
+            [LIBUNWIND_LIB_DIR="$with_libunwind/lib"])
```

Comment:
IIUC, this piece of code means that, if I pass a value for `--with-libunwind` it will be taken as a prefix for the `include` and `lib` directories. For example if I pass "no" or "true"...

Is this really needed?